### ISSUE TITLE
[New formatting analyzer] Add space after single-line comment start

### ISSUE THREAD

**Issue by Evangelink** [dotnet/roslyn#50183]
**Brief description:**

Provide a new analyzer that ensures there is a space after the single line comment start (`//` for C# and `'` for VB.NET).

**Languages applicable:** Both languages

**Code example that the analyzer should report:**

Add something like `csharp_space_after_comment = true`

Which would transform 
```csharp
//Some comment
```

into
```csharp
// Some comment
```

The same is true for VB.NET. I will leave it up to you to decide if the `false` value should simply deactivate the rule or force no space between.

**Additional information:**

We could consider something similar for multiline comments but I think it would need more tweaks so might deserve a separate entry.

**Documentation requirements:**

When this analyzer is implemented, it must be documented by following the steps at [Documentation for IDE CodeStyle analyzers](https://github.com/dotnet/roslyn/blob/master/docs/contributing/Documentation%20for%20IDE%20CodeStyle%20analyzers.md).

**Comment by sharwell** [comment]
Duplicate of #30239
